internal/kafka/internal/handlers/validation.go

```diff
@@ -390,3 +392,13 @@ func ValidateKafkaStorageSize(kafkaRequest *dbapi.KafkaRequest, kafkaUpdateReq *
 		return nil
 	}
 }
+
+func validateKafkaMachinePoolNodeCount(clusterPayload *public.EnterpriseOsdClusterPayload) handlers.Validate {
```

Comment:
should node count be also divisible by 3 with no remainder?

Reply:
That's true. Let me add a check for it as well.